boards/cats_rev1Pro/Core/Src/tasks/task_state_est.c

```diff
@@ -123,12 +122,17 @@ _Noreturn void task_state_est(__attribute__((unused)) void *argument) {
     /* Filter Data */
 #ifdef USE_MEDIAN_FILTER
     median_filter(&filter_data, &state_data);
+    float filtered_acc = 0;
+    for (int i = 0; i < 3; i++) {
+      if (elimination.faulty_imu[i] == 0) {
+        filtered_acc += state_data.acceleration[i] / (float)(elimination.num_faulty_imus);
```

Comment:
Are we now always including the high g accel into the filter?

Reply:
We are including it if we are in high acceleration (larger than the bound), or if its the only remaining working accel

Reply:
Why are we looping over 3 sensors in that case?

Reply:
Oh, so this piece of code is only to compute the average of the filtered data which is then only logged. This variable is not used in the filter at all. Before, I averaged the 3 filter values to save space, maybe now it doesnt make much sense anymore and we should just average the 2 low g accels and log the filtered high g accel on its own.